addons/mail/models/mail_tracking_value.py

```diff
@@ -31,6 +31,9 @@ class MailTracking(models.Model):
     new_value_text = fields.Text('New Value Text', readonly=1)
     new_value_datetime = fields.Datetime('New Value Datetime', readonly=1)
 
+    currency_id = fields.Many2one('res.currency', 'Currency', readonly=1, ondelete='set null',
```

Comment:
readonly=True (1 is for "that kind of developers", you know, "them" ...) .

Reply:
Do I change also the others or just this one ? :)

Reply:
Minimal diff :) so this one only